my_list.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './enumerables'
+
+class MyList
+  include MyEnumerable
+  def initialize(*arg)
+    @list = arg
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
+
+p(list.all? { |e| e < 5 })
+p(list.all? { |e| e > 5 })
+
+p(list.any? { |e| e == 2 })
+p(list.any? { |e| e == 5 })
+
+p(list.filter(&:even?))
```

Comment:
- [ ] Make sure you have implemented the` #each` method that yields all members of `@list `in succession.

Reply:
.Create a module My[Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html) that implements the following methods (they should have the same funcionality as methods in Enumerable):
#all?
#any?
#filter





[4:47](https://microverse-students.slack.com/archives/D039JAVCKFD/p1648738038074989)
each is not in the requirements

Reply:
Please elaborate on this.
The #each method is not part of the project requirements

@arslanbisharat

Reply:
Please check the project requirements. You would find that there. 

![image](https://user-images.githubusercontent.com/58504655/161190258-ca04014e-ba29-471f-8f5a-8827aed388aa.png)